package.json

```diff
@@ -8,9 +8,15 @@
     "*.d.ts"
   ],
   "scripts": {
-    "bundle": "gulp bundle",
-    "doc": "gulp doc",
-    "test": "gulp test"
+    "bundle": "npm run clean && npm run compile && mv _build/src/** . && node _build/tools/bundle.js",
```

Comment:
Doesn't seem much better than a Makefile, tbh. Also, I expect this to be completely unapproachable to someone new -- e.g. what command should I run to do X? It's a shame JSON doesn't support comments.